src/services/tests/category.test.ts

```diff
@@ -0,0 +1,19 @@
+import { describe, expect, jest } from '@jest/globals';
+import categoryRepository from '../../repositories/categoryRepository';
+import categoryService from '../categoryService';
+
+describe('category test', () => {
+  it('should return an object array', async () => {
+    const result = await categoryService.getAll();
+    expect(typeof result).toBe('object');
+  });
+
+  it('should return a not null array', async () => {
+    const result = await categoryService.getAll();
+    expect(result.length).not.toBeLessThanOrEqual(0);
+  });
+  it('should return an error if array is undefined', async () => {
+    const result = await categoryService.getAll();
+    expect(typeof result.length).not.toBe('undefined');
+  });
```

Comment:
O nome desse teste sugere que o getAll lança um erro quando sua busca não retorna resultado, acredito que seria interessante adicionar esse erro na função ou alterar o nome dela para algo como "should return a defined result"

Reply:
mudanças feitas